src/cmd-prune-pkgcache

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+print_help() {
+    cat 1>&2 <<'EOF'
+Usage: coreos-assembler prune-pkgcache --help
+       coreos-assembler prune-pkgcache
+
+  Prune refs packages from the pkgcache, which are not present in the builds and modified over 30 days.
+EOF
+}
+
+rc=0
+options=$(getopt --options h --longoptions help -- "$@") || rc=$?
+[ $rc -eq 0 ] || {
+    print_help
+    exit 1
+}
+eval set -- "$options"
+while true; do
+    case "$1" in
+        -h | --help)
+            print_help
+            exit 0
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *)
+            fatal "$0: unrecognized option: $1"
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+if [ $# -ne 0 ]; then
+    print_help
+    fatal "ERROR: Too many arguments"
+    exit 1
+fi
+
+#Convert all the builds pkg names to the naming convention
+bdspkgstr=$(jq '."rpmostree.rpmdb.pkglist"' "$(find /srv/builds/ -name "commitmeta.json")" \
+	| jq -rM '.[] | (.[0] + "/" + (if .[1] != "0" then (.[1] + "_3A") else null end)
+	+ (.[2:4] | join("-")) + "." + (if .[4] == "x86_64" then "x86__64" else .[4] end))')
```

Comment:
I would consider the layout of the rpm-ostree package cache to *mostly* be an implementation detail of rpm-ostree subject to change (though, changing it in any kind of incompatible way risks backcompat issues).

In this case...sorry I hadn't looked at this topic before or I probably would have recommended we implement this in rpm-ostree, something like `rpm-ostree compose prune-pkgcache --repo=cache/pkgcache-repo` and then this command just calls it.

But, since this is a really special developer-only case I'm OK shipping an initial implementation in coreos-assembler; we can always "lower" it later.

That said, this specific code...I think we should probably implement the quoting handling in Python or Go.  It's basically reimplementing `rpmostree_nevra_to_cache_branch()` and `rpmostree_cache_branch_to_nevra()`.

Reply:
I noticed `cosa prune` command is written by Python. So, I reimplemented the code by Python, and put into the `cmd-prune` part as `cosa prune --pkgcache`.

Reply:
Sounds good!